spec/requests/admin/configs_spec.rb

```diff
@@ -237,13 +237,31 @@
           expect(SiteConfig.main_social_image).to eq(expected_image_url)
         end
 
+        it "only updates the main_social_image if given a valid image URL" do
+          expected_image_url = "![logo_lowres]https://dummyimage.com/300x300"
+          expect do
+            post "/admin/config", params: { site_config: { main_social_image: expected_image_url },
+                                            confirmation: confirmation_message }
+            # expect(SiteConfig.main_social_image).not_to eq(expected_image_url)
```

Comment:
I'll remove this comment in addition to the comments on lines  `261`, `279`, and `302` once I have the validation in the controller working!